### PR TITLE
Add dialog editor

### PR DESCRIPTION
**Pull request by amanmahajan7**
image

We should probably still render the cell otherwise it is empty

### CODE REVIEW

File: stories/demos/CommonFeatures.tsx [hunk 1]
```diff
@@ -13,6 +13,28 @@ const toolbarClassname = css`
   margin-bottom: 8px;
 `;
 
+const dialogContainerClassname = css`
```
Comment (amanmahajan7) [inline stories/demos/CommonFeatures.tsx:16]: > The ::backdrop CSS pseudo-element can be used to style behind a <dialog> element when the dialog is displayed with HTMLDialogElement.showModal().

😞

Comment (nstepien) [reply stories/demos/CommonFeatures.tsx:16]: Would that work? 🤔 
```tsx
ref={dialog => dialog.showModal()}
```

**Review by nstepien — APPROVED** [review]

**Comment by nstepien** [comment]
>We should probably still render the cell otherwise it is empty

Let's do that yeah, should be easy with the grid layout.

File: stories/demos/CommonFeatures.tsx (hunk 1 above)
Comment (amanmahajan7) [reply stories/demos/CommonFeatures.tsx:16]: Tried but getting ref errors. May be I will try some other time

**Pull request merged**